ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/query/QueryExecutorTest.java

```diff
@@ -194,7 +195,7 @@ public void setUp() {
     queryExecutor = new QueryExecutor(ksqlEngine, ksqlRestConfig, ksqlConfig,
         Optional.of(pullQueryExecutorMetrics), Optional.of(scalablePushQueryMetrics), rateLimiter,
         concurrencyLimiter, pullBandRateLimiter, scalablePushBandRateLimiter,
-        haRouting, pushRouting, Optional.of(localCommands));
+        haRouting, pushRouting, Duration.ofMillis(100), Optional.of(localCommands));
```

Comment:
why this is needed?

Reply:
Oh, thanks for catching that! This was for testing, this is the poll timeout for the kafka consumer and we have that info in the restConfig. 

`ksqlRestConfig.getLong(KsqlRestConfig.STREAMED_QUERY_DISCONNECT_CHECK_MS_CONFIG)`

I'll push the change.